documentation/MeetingMinutes/MOM-2023-09-06.md

```diff
@@ -0,0 +1,101 @@
+| [#45](https://github.com/camaraproject/IdentityAndConsentManagement/issues/45) ([PR#55](https://github.com/camaraproject/IdentityAndConsentManagement/pull/55)) | Deutsche Telekom AG | **Add missing details in authN-authZ doc**<br>- See notes of review of PR #55. |
+| [#46](https://github.com/camaraproject/IdentityAndConsentManagement/issues/46) | Deutsche Telekom AG | **Extend AuthN-AuthZ doc with detailed CIBA flow description**<br>- Was closed when PR #54 was merged, reopened to allow Shilpa (Deutsche Telekom) to include reference in AUthN-AuthZ document pointing to CIBA flow.  Kept open until Shilpa is back is back. | 
+| **Closed issues** |  |  |
+| [#15](https://github.com/camaraproject/IdentityAndConsentManagement/issues/15) | GSMA/Vonage | **Subscriber Identity API**<br> Was closed as agreed on 23/8 call as there were no further comments nor objections. |
+| [#52](https://github.com/camaraproject/IdentityAndConsentManagement/issues/52) ([PR#54](https://github.com/camaraproject/IdentityAndConsentManagement/pull/54)) | GSMA | **Differences in authentication on APIs between deployments can lead to differences in API usage and break federation**<br> Closed after merge of PR #54 |
+| **New issues** |  |  |
+| [#57](https://github.com/camaraproject/IdentityAndConsentManagement/issues/57) ([PR#33](https://github.com/camaraproject/IdentityAndConsentManagement/pull/33))| Telefonica | **Align securitySchemes and security of CAMARA API specs with IdentityAndConsentManagement WG**<br> - Will progress once Commonalities PR is merged. Discussion will continue offline. |
```

Comment:
I am not sure if the issue is related to PR#33.
```suggestion
| [#57](https://github.com/camaraproject/IdentityAndConsentManagement/issues/57) ([PR#33](https://github.com/camaraproject/IdentityAndConsentManagement/pull/33))| Telefonica | **Align securitySchemes and security of CAMARA API specs with IdentityAndConsentManagement WG**<br> - Will progress once [Commonalities](https://github.com/camaraproject/Commonalities/pull/57) PR  is merged. Discussion will continue offline. |
```

Reply:
Yes, that was wrong. Actually that issue does not have a related PR yet. Removing that and updating the link to commonalities PR in this commit: https://github.com/camaraproject/IdentityAndConsentManagement/pull/60/commits/94c9f9ea9a91294f70b985d2fdbb69239018e46a